src/elastiCache.js

```diff
@@ -0,0 +1,104 @@
+const redis = require('ioredis');
+const _ = require('lodash');
+
+// uncomment the line below if you want to use moment to set the cache deletion time
+// const moment = require('moment-timezone');
+
+// AWS ElastiCache with redis cluster mode
+// uncomment the lines below and replace the host with your configuration endpoint
+// change lazyConnect to true if you want redis client to connect to the redis server as soon as it's created
+// with lazyConnect: true, it will only connect when running query against the redis database
+// const redisClient = new redis.Cluster([
+//   {host: 'your-elastiCache-configuration-endpoint', port: 6379, lazyConnect: true}
+// ]);
+
+// AWS ElastiCache with redis cluster mode OFF
+// uncomment the lines below and replace the host with your node endpoint
+// change lazyConnect to true if you want redis client to connect to the redis server as soon as it's created
+// with lazyConnect: true, it will only connect when running query against the redis database
+const redisClient = new redis({host: 'localhost', port: 6379, lazyConnect: true});
```

Comment:
Not a big issue, but would it be helpful to make the host configurable?

Reply:
added dotenv, now the user can config cache mode, elastiCache endpoint and lazy connect in the .env file